x-pack/plugins/spaces/public/space_avatar/space_avatar_internal.tsx

```diff
@@ -20,6 +20,7 @@ interface Props {
   size?: 's' | 'm' | 'l' | 'xl';
   className?: string;
   announceSpaceName?: boolean;
+  isDisabled?: boolean;
```

Comment:
Again, not a problem you introduced, but would you mind adding a comment for at least this new `isDisabled` field?